src/motile_tracker/data_views/views/ortho_views.py

```diff
@@ -0,0 +1,263 @@
+import inspect
+import time
+
+import napari_orthogonal_views.ortho_view_widget as ov_widget
+from napari.layers import Labels, Layer, Points, Shapes
+from napari.utils.events import Event
+from napari.utils.notifications import show_info
+from napari_orthogonal_views.ortho_view_manager import _get_manager  # noqa
+
+from motile_tracker.data_views.views.layers.track_graph import TrackGraph
+from motile_tracker.data_views.views.layers.track_labels import TrackLabels
+from motile_tracker.data_views.views.layers.track_points import TrackPoints
+
+
+# redefinition of copy_layer function
+def copy_layer(layer: Layer, name: str = ""):
+    if isinstance(
+        layer, TrackGraph
+    ):  # instead of showing the tracks (not very useful on 3D data because they are
+        # collapsed to a single frame), use an empty shapes layer as substitute to ensure
+        # that the layer indices in the orthogonal viewer models match with those in the
+        # main viewer
+        res_layer = Shapes(
+            name=layer.name,
+            data=[],
+        )
+
+    elif isinstance(layer, TrackLabels):
+        res_layer = Labels(
+            data=layer.data,
+            name=layer.name,
+            opacity=layer.opacity,
+            scale=layer.scale,
+            colormap=layer.colormap,
+        )
+
+    elif isinstance(layer, TrackPoints):
+        res_layer = Points(
+            data=layer.data,
+            name=layer.name,
+            symbol=layer.symbol,
+            face_color=layer.face_color,
+            size=layer.size,
+            properties=layer.properties,
+            border_color=layer.border_color,
+            scale=layer.scale,
+            blending="translucent_no_depth",
```

Comment:
Might actually be called blending_mode
```suggestion
            blending=layer.blending,
```

Reply:
done!

Reply:
This already bit me when I was using the branch for experiments 😆 happy to have it fixed!